.github/workflows/linux-ci.yml

```diff
@@ -57,19 +72,37 @@ jobs:
           cache: poetry
 
       - name: Install python deps
-        run: poetry install -E core -E backend
-
-      - name: Configure libfuse path
-        run: |
-          LIBFUSE_PATH=$(find /usr/lib -name '*fuse*.so*' 2>/dev/null | head -n 1)
-          echo "::warning title=Libfuse path::use lib found at $LIBFUSE_PATH"
-          echo "FUSE_LIBRARY_PATH=$LIBFUSE_PATH" >> "$GITHUB_ENV"
+        run: poetry install -E core -E backend --remove-untracked
 
       - name: Generate PyQT GUI data
         run: poetry run python misc/generate_pyqt.py
 
       - name: Install libparsec_python
+        if: matrix.binding == 'rust-binding'
         run: poetry run pip install oxidation/libparsec_python
 
-      - name: Test python codebase
-        run: poetry run pytest tests --runrust
+      - name: Basic tests
+        run: timeout 200 poetry run pytest $PYTEST_BASE_ARGS tests
+
+      - name: Rust binding tests
+        if: matrix.binding == 'rust-binding'
+        run: timeout 200 poetry run pytest $PYTEST_BASE_ARGS tests --runrust -m rust
+
+      - name: Mountpoint tests
+        run: timeout 200 poetry run pytest $PYTEST_BASE_ARGS tests --runmountpoint --runslow -m mountpoint
+
+      - name: GUI tests
+        run: |
+          # Required dependencies for Qt testing. See:
+          # https://pytest-qt.readthedocs.io/en/latest/troubleshooting.html#github-actions
+          sudo apt-get install libxkbcommon-x11-0 libxcb-icccm4 libxcb-image0 libxcb-keysyms1 libxcb-randr0 libxcb-render-util0 libxcb-xinerama0 libxcb-xfixes0 x11-utils
```

Comment:
from the link, you're missing the line

```bash
/sbin/start-stop-daemon --start --quiet --pidfile /tmp/custom_xvfb_99.pid --make-pidfile --background --exec /usr/bin/Xvfb -- :99 -screen 0 1920x1200x24 -ac +extension GLX
```

It's expected ?

Reply:
> you're missing the line [...]

This is for Travis CI as far as I understand.

Reply:
I'm counting on `pytest-xvfb` to deal with this kind of stuff, and it seems to work so far